pkg/features/features.go

```diff
@@ -79,6 +81,7 @@ var (
 		FeatureCloudConfigDefaults,
 		FeatureCloudConfigEssentials,
 		FeatureBootAssessment,
+		FeatureArmFirmware,
```

Comment:
I wonder if we should actually include it here or not. It does not have much impact on Elemental as we are not using the default value on our Micro images, however it has on the example images. If not included here for the RPi example we will have to list all features in the command line 🤔 

Probably would make sense having some sort of `--all` flag to force the installation of all features, assuming that default ones is a different set. Any thoughts?

Reply:
I would also not include this as the default, unless we check that the target platform actually is `arm`..

A good trade-of would be as you mentioned to add the `all` value to include all features [here](https://github.com/rancher/elemental-toolkit/blob/b4765b9fc1b4048c643e1475893514e796240ed3/cmd/init.go#L55)

Reply:
For now I did not add the `--all` flag andremoved the `arm-firmware` from the defaults list. However I thought about adding features as arguments instead of a single comma separated value. With this we can easily list them in a Dockerfile as a multi-line arguments list. This way is also possible to use the  `cobra.OnlyValidArgs` method to ensure the provided features exist. Moreover, it is easy to keep backward compatibility, hence we can actually  change this `elemental init` command UX without actually breaking any existing Dockerfile.

Reply:
Yep, that looks good! Nice job!